### ISSUE TITLE
Stray symlink named "{" getting created in 2.6.1

### ISSUE THREAD

**Issue by kpaulisse** [rodjek/rspec-puppet#542]
Puppet 4.10.0 and rspec-puppet 2.6.1. When running any rspec-puppet test, I get a symlink created that looks like this:

```
$ ls -l spec/fixtures/modules/\{
spec/fixtures/modules/{ -> (root of my puppet checkout)
```

Via some code hacking I discovered that this was getting the module name of `{` from https://github.com/rodjek/rspec-puppet/blob/master/lib/rspec-puppet/setup.rb#L63-L67 as it parsed the first file it found in the manifests directory, which looks like this:

```
node /^a-regex-matching-the-host-name$/ {
  class { 'foo::bar::baz':
    (various parameters)
  }
}
```

The module_name = "{" is coming from parsing a file that looks like the above. This particular file is just the first file in our "manifests" directory, but it has nothing to do with the class being tested.

The thing that seems to have changed between the versions is the addition of this call: https://github.com/rodjek/rspec-puppet/blob/v2.6.1/lib/rspec-puppet.rb#L42-L44 -- I don't see this problem (or any symlinks) being created in 2.5.0.

This "{" symlink does not get cleaned up after the test.

If I go ahead and comment out lines 42-44, the symlink does not get created, and the test still passes. However I'm hesitant to submit this as a patch because I don't know the intention of those lines, so I chose to submit it as an issue instead.

**Comment by rodjek** [comment]
Yeah, that before suite hook is there for setting up the test environment for an individual module. I forgot about the case of people running rspec-puppet from their control repo. I'll fix it up in the 2.6.2 release tonight.

**Comment by rodjek** [comment]
rspec-puppet 2.6.2 pushed to rubygems
